### PR TITLE
New module: gce_tag

### PR DESCRIPTION
**Pull request by dohoangkhiem**
New module named 'gce_tag' for add/remove tags to/from GCE instance

### CODE REVIEW

**Comment by bcoca** [comment]
cc @erjohnso

File: cloud/google/gce_tag.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python
+# Copyright 2015 Google Inc. All Rights Reserved.
```
Comment (erjohnso) [inline cloud/google/gce_tag.py]: Do you mean to assign copyright to Google? You can/should associate this with yourself or your employer (if required).

File: cloud/google/gce_tag.py [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+        module.fail_json(msg='Must specify "zone"', changed=False)
+
+    if not tags:
+        module.fail_json(msg='Must specify "tags"', changed=False)
+
+    gce = gce_connect(module)
+
+    # add tags to instance.
+    if state == 'present':
+        results = add_tags(gce, module, instance_name, tags)
```
Comment (erjohnso) [inline cloud/google/gce_tag.py]: Nit: You could make this one line (similarly with line 223-225),

``` python
changed, tags_changed = add_tags(gce, module, instance_name, tags)
```

**Comment by erjohnso** [comment]
Thanks for this @dohoangkhiem!  I left a few small comments, but overall this is awesome and a welcome addition!

**Comment by dohoangkhiem** [comment]
Thanks for review @erjohnso ! , I made a commit fixed those minor issues.

**Comment by erjohnso** [comment]
Thanks @dohoangkhiem !

**Pull request merged**